README.md

```diff
@@ -39,20 +39,68 @@ on:
 
 jobs:
   autodev:
-    uses: Staffbase/gha-workflows/.github/workflows/template_autodev.yml@<version>
+    uses: Staffbase/gha-workflows/.github/workflows/template_autodev.yml@v1.3.0
+    with:
+      # optional: base branch from which the history originates, default: main
+      base: master
+      # optional: name of the dev branch, default: dev
+      branch: your dev branch
+      # optional: update status comment, default: false
+      # if you want to change the message, please adapt 'success_comment' and/or 'failure_comment'
+      comments: true
+      # optional: update status label, default: false
+      # if you want to change the labels, please adapt 'success_label' and/or 'failure_label'
+      labels: true
+      # optional: label which should trigger the action, default: dev
+      label: deploy
+      # optional: name of the user which does the commit, default: AutoDev Action
+      user: your name
+      # optional: mail of the user which doe the commit, default: staffbot@staffbase.com
```

Comment:
Axel mentioned to me earlier that we should refrain from using staffbase parts in here. Therefore:  `bot@example.com`

Reply:
Yes I thought about that too, but since the email is used in other public repos, I left it as default.